sdk/identity/identity/test/public/node/azurePipelinesCredential.spec.ts

```diff
@@ -44,14 +44,44 @@ describe("AzurePipelinesCredential", function () {
       systemAccessToken,
     );
     const regExp: RegExp =
-      /AzurePipelinesCredential: Authenticated Failed. Received null token from OIDC request. Response status- 404./;
+      /invalid_client: Error\(s\): 700213 .* AADSTS700213: No matching federated identity record found for presented assertion subject .* Please note that the matching is done using a case-sensitive comparison. Check your federated identity credential Subject, Audience and Issuer against the presented assertion/;
     await assert.isRejected(
       credential.getToken(scope),
       regExp,
       "error thrown doesn't match or promise not rejected",
     );
   });
 
+  it("failure includes the expected response headers", async function () {
+    if (!isLiveMode()) {
+      this.skip();
+    }
+    // clientId for above service connection
+    const clientId = process.env.AZURE_SERVICE_CONNECTION_CLIENT_ID!;
+    const existingServiceConnectionId = process.env.AZURE_SERVICE_CONNECTION_ID!;
+    // const systemAccessToken = process.env.SYSTEM_ACCESSTOKEN!;
+    const credential = new AzurePipelinesCredential(
+      tenantId,
+      clientId,
+      existingServiceConnectionId,
+      "systemAccessToken",
```

Comment:
```suggestion
      "invalidSystemAccessToken",
```

Reply:
Just expresses the intent here a little better, but this is a totally minor nitpick 😄